content/cpp/concepts/math-functions/terms/round/round.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'round()'
+Description: 'Returns the integral value that is nearest to the argument, with halfway cases rounded away from zero.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `round()` function returns the integral value that is nearest to the argument, with halfway cases rounded away from zero.
+
+## Syntax
+
+The `cmath` library must be added at the top of the file.
+
```

Comment:
```suggestion

```

I would omit this line since this point is covered in the parent [math functions entry](https://www.codecademy.com/resources/docs/cpp/math-functions).